### ISSUE TITLE
How works CPU constant mode (when RAPL files are not found) ? 

### ISSUE THREAD

**Issue by Sarah7223** [mlco2/codecarbon#477]
Hello !

I am trying to use CodeCarbon on a Linux virtual machine. I don't have the RAPL module on it so the cpu tracking falls on constant mode.
If I understood well the methodology, in this case the cpu energy consumption = TDP * usage (constante=0,5) ?
If it's true, experimentally, the cpu_energy divided by the duration should be constant between different measures on the same machine. In practice, I made several measures on the same machine which are saved on a .csv, and then I calculated cpu_ernergy/duration on all the lines, I get close values but not equal, do you know why ? 

I saw that you began to work on CPU load tracking with psutil, it is not merged yet, right ? 
Do you know if it is possible to custom the kernel of a virtual machine so it is able to use RAPL files or it doesn't work on virtual environnements ? 

Thank you for your help !

**Comment by benoit-cty** [comment]
Sorry, I dont think it could work with Virtual Machine.
With containers, like Docker, you could allow the host to share system files with the host.

Yes the CPU load is coming and you could try the code of the branch if you want, it's already supposed to work.

**Comment by RafiullahOmar** [comment]
@benoit-cty 
When we get "No CPU tracking mode found. Falling back on CPU constant mode." message.  Do we get the accurate data with CPU constant mode?

**Comment by benoit-cty** [comment]
It's a constant and do not consider the CPU usage, so it is not accurate.

**Comment by RafiullahOmar** [comment]
I believe that since the 'Test under Linux with RAPL' has been done in #316, so even if we encounter the warning 'No CPU tracking mode found. Falling back on CPU constant mode,' we will still obtain accurate results. am I right?

**Comment by benoit-cty** [comment]
Yes the goal of this PR is to use the CPU load in real time and multiply it to the TDP.
